changelog.rst

```diff
@@ -4,6 +4,9 @@ Compysition changelog
 Version
 1.3.0
 
+- Adjusted ContentTypePlugin to properly validate Content-Type headers.
+    - Now validates all incoming requests vs only the first.
+    - Now instantly triggers HTTP errors when met with requests containing missing/empty Content-Type headers and have a non-empty body.
```

Comment:
Can this behavior be changed easily? We very well might have third-parties that are sending us empty content-type's with a body, wondering how easy this would be to revert and allow.

Reply:
It would be a one line change/removal.